src/auth/mod.rs

```diff
@@ -8,24 +10,47 @@ pub(crate) mod slack;
 pub(crate) mod telegram;
 pub mod utils;
 
-pub const COOKIE_TOKEN: &str = "access_token";
+pub const COOKIE_TOKEN: &str = "token";
+//pub const USER_DETAIL: &str = "ud";
+pub const GITHUB_PROVIDER: &str = "github";
+pub const TELEGRAM_PROVIDER: &str = "telegram";
+pub const DISCORD_PROVIDER: &str = "discord";
+pub const SLACK_PROVIDER: &str = "slack";
+pub const GOOGLE_PROVIDER: &str = "google";
```

Comment:
Use enum for providers
```suggestion

pub enum AuthProviders {
     GITHUB,
     TELEGRAM,
     GOOGLE,
     DISCORD,
     SLACK
}
```

Reply:
done